mappings/net/minecraft/predicate/entity/EntityPredicates.mapping

```diff
@@ -9,3 +9,4 @@ CLASS aic net/minecraft/predicate/entity/EntityPredicates
 	FIELD f EXCEPT_SPECTATOR Ljava/util/function/Predicate;
 	METHOD a maximumDistance (DDDD)Ljava/util/function/Predicate;
 		ARG 0 x
+	METHOD b getBottommostEntity (Lahz;)Ljava/util/function/Predicate;
```

Comment:
`getBottommostEntityChecker`. See the return type `java.util.function.Predicate`